src/witan/models/load_data.clj

```diff
@@ -45,7 +45,7 @@
   (make-ordered-ds-schema [[:gss-code s/Str] [:district s/Str] [:sex s/Str] [:age s/Int]
                            [:var s/Str] [:year s/Int] [:estimate s/Num]]))
 
-(def HistPopnEst
+(def BasePopn
```

Comment:
Is it necessary to repeat this schema here and https://github.com/MastodonC/witan.models.demography/pull/18/files#diff-e7253d5ba9334b4ad5edd60a2aa084b7R14 ?